src/module.js

```diff
@@ -59,33 +59,21 @@ var debug = {
     FeatureSelection
 };
 
-// Window can only be set in main thread
-if (Utils.isMainThread) {
-
-    window.Tangram = module.exports = {
-        leafletLayer,
-        debug,
-        version: version.string
-    };
-
-}
-
-if (Utils.isWorkerThread) {
```

Comment:
So actually, as you originally mentioned, simply exporting Tangram will attach it to the worker, and so code block can be safely deleted.

It's a little confusing to me but I guess the worker loads the tangram source [here](https://github.com/tangrams/tangram/blob/master/src/scene.js#L255-L267)?